ocw/lib/db.py

```diff
@@ -161,7 +161,7 @@ def _update_provider(name, vault_namespace):
         for region in EC2(vault_namespace).all_regions:
             instances_csp = EC2(vault_namespace).list_instances(region=region)
             instances += [ec2_to_local_instance(i, vault_namespace, region) for i in instances_csp]
-            logger.info("Got %d instances from EC2 in region %s", len(instances), region)
+        logger.info("Got %d instances from EC2", len(instances))
```

Comment:
Why? I would keep it more verbose.

Reply:
1. because it is a fake , if you look closely you will see that we sum together all entities in one value but in log you see like each region has that amount of entities 
2.  from 2 years personal experience reading logs I don't need such details per region , total amount of entities is more than enough